src/mlpack/core/optimizers/cmaes/test.cpp

```diff
@@ -0,0 +1,69 @@
+/**
+ * @file test.cpp
```

Comment:
We should put the test in ``mlpack/tests`` and use the boost test framework to run the test. Take a look at the existing tests in ``mlpack/tests`` for examples.